src/ensembl/src/shared/components/feature-summary-strip/FeatureSummaryStrip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { GeneSummaryStrip, RegionSummaryStrip } from '../feature-summary-strip';
+
+import { EnsObject } from 'src/shared/state/ens-object/ensObjectTypes';
+
+type FeatureSummaryStripProps = {
+  ensObject: EnsObject | null;
+  isDrawerOpened: boolean;
```

Comment:
As a component, `FeatureSummaryStrip` does not need to know anything about the drawer — it's not its responsibility. Since the children components have an `isGhosted` prop, I suggest that `FeatureSummaryStripProps` also includes `isGhosted`, which will be set by the parent component, which will be aware of the drawer.

Reply:
I think its just a naming problem if i understand the code correctly; further down isDrawerOpened  is then assigned to isGhosted which whats needed as you suggested. I have just renamed the variable to isGhosted, that should be fine isn't it?